src/1729Essay.sol

```diff
@@ -65,35 +96,13 @@ contract OneSevenTwoNineEssay is Ownable, ERC721 {
         return tokenId;
     }
 
-    /// @notice Returns the total of all tokens ever minted (includes tokens which have been burned)
-    function totalSupply() public view returns (uint256) {
-        return nextTokenId.current() - 1;
+    function burn(uint256 tokenId) public virtual onlyOwner {
+        _burn(tokenId);
```

Comment:
@plaird523 Do we need still this as a separate internal function, or can we move the body of _burn() into burn() ?

Reply:
We can, but be sure to preserve the delete essays[tokenId] line.